doc/api/repl.md

```diff
@@ -147,39 +147,6 @@ global or scoped variable, the input `fs` will be evaluated on-demand as
 > fs.createReadStream('./some/file');
 ```
 
-#### Global uncaught exceptions
-
-<!-- YAML
-changes:
-  - version: v12.3.0
-    pr-url: https://github.com/nodejs/node/pull/27151
-    description: The `'uncaughtException'` event is from now on triggered if the
-                 repl is used as standalone program.
--->
-
-The REPL uses the [`domain`][] module to catch all uncaught exceptions for that
-REPL session.
-
-This use of the [`domain`][] module in the REPL has these side effects:
-
-* Uncaught exceptions only emit the [`'uncaughtException'`][] event in the
-  standalone REPL. Adding a listener for this event in a REPL within
-  another Node.js program results in [`ERR_INVALID_REPL_INPUT`][].
```

Comment:
I'd replace this section with one that describes the new behavior and amend the `changes:` YAML entry instead of dropping it (since the new behavior is definitely one that doesn't matter to some users but will have a significant negative impact on users who rely on it)

Reply:
I don't think a specific section is needed, IMO there's nothing to document. it adds a listener to the process object, what negative impact does that have?

Reply:
@RedYetiDev Well, before this change, `uncaughtException` listeners wouldn't have seen uncaught exceptions coming from the REPL, and now they do, right? It's not a massive change or anything, but I'd still document it _somewhere_. If you feel that no separate section is necessary, I'd still keep the `changes:` block to document the change in behavior over time

Reply:
I am confused about this change. As far as I see it in the code, only the stand alone repl works with the uncaughtException listeners. In case more than a single REPL instance is running, it is not possible to identify the correct listeners, so none receive the errors.